modules/extensions-support/src/main/java/org/mule/module/extension/internal/runtime/config/LifecycleAwareConfigurationInstance.java

```diff
@@ -165,6 +166,7 @@ private void doInitialise() throws InitialisationException
         if (connectionProvider.isPresent())
         {
             initialiseIfNeeded(connectionProvider, true, muleContext);
+            initialiseIfNeeded(connectionManager, true, muleContext);
```

Comment:
Mule will initialise the connection manager automatically. You shouldn't do this here